app/lib/package/backend.dart

```diff
@@ -433,7 +433,7 @@ class GCloudPackageRepository extends pub_server.PackageRepository {
   }
 
   @override
-  Future<pub_server.AsyncUploadInfo> startAsyncUpload(Uri redirectUrl) async {
+  Future<api.AsyncUploadInfo> startAsyncUpload(Uri redirectUrl) async {
```

Comment:
Let's rename this method and call it something else.

Reply:
Maybe `startUpload`?

Reply:
(merging this, we can do the rename later if `startUpload` is not the best one...)

Reply:
Or `getUploadUrlInfo` or similar...